LFS171x/fabric-material/tuna-app/README.md

```diff
@@ -1,9 +1,23 @@
-## Hyperledger Fabric Sample Application
+## Hyperledger Fabric Sample Application (Tuna App)
 
-This application demonstrates the creation and transfer of tuna fish shipments between actors leveraging Hyperledger Fabric in the supply chain. In this exercise we will set up the minimum number of nodes required to develop chaincode. It has a single peer and a single organization.
+    This application demonstrates the creation and transfer of tuna fish shipments between actors leveraging Hyperledger Fabric in the supply chain. In this exercise we will set up the minimum number of nodes required to develop chaincode. It has a single peer and a single organization.
+
+## Installation
+
+1. Start the Fabric network ``./startFabric.sh``
+
+2. Install the npm packages ``npm install``
+
+3. Register the Admin ``node registerAdmin.js``
+
+4. Register the User ``node registerUser.js``
+
+5. Run the server ``node server.js``
+
+6. View at browser ``localhost:8000``
 
 if getting error about running ./startFabric.sh permission 
```

Comment:
It would be worthwhile to move this troubleshooting information (lines 19-21) so that it exists under bullet point 1.